link_linux.go

```diff
@@ -956,13 +956,13 @@ func LinkSetXdpFdWithFlags(link Link, fd, flags int) error {
 	return err
 }
 
-// LinkSetGSOMaxSize sets the GSO maximum size of the link device.
+// LinkSetGSOMaxSize sets the IPv6 GSO maximum size of the link device.
```

Comment:
Is this true for the iproute2 command as well, that it is for ipv6 only?

Also, I do not find the gso_ipv4_max_size option in iproute2 shipped in RHEL 9. 

Is this a very recent addition?

Reply:
BIG TCP was initially targeted only for IPv6-only: https://lore.kernel.org/netdev/20220513183408.686447-3-eric.dumazet@gmail.com/T/  . This went into 5.19 kernel.

Only later IPv4 BIG TCP got added in 6.3 kernel: https://lore.kernel.org/netdev/cover.1674921359.git.lucien.xin@gmail.com/

I doubt this is anywhere in RHEL at this point given it's fairly recent.

Reply:
Forgot to add the iproute2 link: https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=1dafe448c7a2f2be5dfddd8da250980708a48c41

@aboch let me know if you need any other info.

Reply:
Thank you @borkmann that's the context I needed.